### DISCUSSION TITLE
Datepicker

### DISCUSSION THREAD

**Discussion by aliqasemzadeh** [saadeghi/daisyui#2811] [Ideas / feature requests]
Hi
I think we need a datepicker that supports Jalali and other calendars.

**Comment by saadeghi** [comment]
daisyUI is a CSS library.  
There are many date picker packages. You can add daisyUI colors to them so they would fit the current theme.

**Comment by aliqasemzadeh** [comment]
@saadeghi 
Which one do you offer for daisyui?

**Reply by saadeghi** [reply]
For which framework?

**Reply by aliqasemzadeh** [reply]
I am using alpinejs. and I need jalali datepicker.
Note:I want to use it as offline.
